CMake/HPHPFindLibs.cmake

```diff
@@ -81,10 +81,14 @@ endif()
 set(CMAKE_REQUIRED_LIBRARIES)
 
 # libXed
-find_package(LibXed)
-if (LibXed_INCLUDE_DIR AND LibXed_LIBRARY)
-  include_directories(${LibXed_INCLUDE_DIR})
+if (NOT DISABLE_XED)
+  find_package(LibXed)
+  if (LibXed_FOUND)
+    include_directories(${LibXed_INCLUDE_DIR})
+  endif()
   add_definitions("-DHAVE_LIBXED")
```

Comment:
- you probably want this when using third-party
- it's currently being defined even when xed is not found, and third-party is not available, leading to build failrues

Reply:
In summary, if the system libxed was not validated and there's no third-party directory, the build should not fail? I could add checks using `if(TARGET xed)` to verify that there's a third-party xed available, but this is not a pattern widely used by other third party libraries. Do you think that's the way to go?